Deploy-CAM.ps1

```diff
@@ -1152,121 +1151,46 @@ function New-ConnectionServiceDeployment() {
     $kvName = $keyVault.Name
     # TODO - make sure user account has keyvault secret access here. Try to add self if not
     # and if doesn't work, probably fail.
-    $generatedDeploymentParameters = @"
-{
-	"`$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
-	"contentVersion": "1.0.0.0",
-	"parameters": {
-		"CSUniqueSuffix": {
-			"value": "$CSsuffix"
-		},
-		"domainAdminUsername": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "domainAdminUsername"
-			}
-		},
-		"domainAdminPassword": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "domainJoinPassword"
-			}
-		},
-		"domainName": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "domainName"
-			}
-		},
-		"LocalAdminUsername": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "connectionServiceLocalAdminUsername"
-			}
-		},
-		"LocalAdminPassword": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "connectionServiceLocalAdminPassword"
-			}
-		},
-		"CSsubnetId": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "connectionServiceSubnet"
-			}
-		},
-		"GWsubnetId": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "gatewaySubnet"
-			}
-		},
-		"CAMDeploymentBlobSource": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "CAMDeploymentBlobSource"
-			}
-		},
-		"certData": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "CAMCSCertificate"
-			}
-		},
-		"certPassword": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "CAMCSCertificatePassword"
-			}
-		},
-		"remoteWorkstationDomainGroup": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "remoteWorkstationDomainGroup"
-			}
-		},
-		"CAMDeploymentInfo": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "CAMDeploymentInfo"
-			}
-		},
-		"_baseArtifactsLocation": {
-			"reference": {
-				"keyVault": {
-					"id": "$kvID"
-				},
-				"secretName": "artifactsLocation"
-			}
-		}
-	}
-}
-"@
+
+    $csRGName = $null
```

Comment:
Maybe add a comment for what the while loop does?

Reply:
Good point. Done (in my other working branch.)